lib/symlinks.js

```diff
@@ -1,19 +1,27 @@
 const gutil = require('gulp-util');
+const minimist = require('minimist');
 const fs = require('fs');
 const path = require('path');
 const paths = require('./paths');
 
 const suffix = '.original';
+const options = minimist(process.argv.slice(2), {
```

Comment:
I'm basically duplicating this from the gulpfile. Is there a better way to pass an argument? This seems ok, but maybe there's a better way?

Reply:
parse the options in gulpfile.js. Pass the arguments to symlinks.task.setup and symlinks.task.teardown.

Reply:
gulp tasks do not accept arguments (see the setup task) so you'll need to create a helper task. From the symlink file, export functions that take arguments. In the gulpfile.js, wrap them in a function and then use that in dependencies:

```js
function symlinkSetupTask() {
  return symlinks.setup(options.symlinks);
}
function symlinkTeardownTask() {
  return symlinks.teardown(options.symlinks);
}
```

Then when defining tasks:
```js
gulp.task('setup-symlinks', symlinkSetupTask);
...
```